L/LLVM/common.jl

```diff
@@ -15,12 +15,25 @@ const llvm_tags = Dict(
     v"12.0.1" => "980d2f60a8524c5546397db9e8bbb7d6ea56c1b7", # julia-12.0.1-4
     v"13.0.1" => "8a2ae8c8064a0544814c6fac7dd0c4a9aa29a7e6", # julia-13.0.1-3
     v"14.0.6" => "ad0184c1a2ee9508319db7c97d635a75d5336cf8", # julia-14.0.6-2
+    v"15.0.7" => "90e1c629f21fe3a8d66d06ebdc5dde62aa6203be", # julia-15.0.7-0
 )
 
 const buildscript = raw"""
 # We want to exit the program if errors occur.
 set -o errexit
 
+if [[ ("${target}" == x86_64-apple-darwin) && ("${LLVM_MAJ_VER}" -ge "12") ]]; then
```

Comment:
I think we might be missing a `*` here to match the Darwin version:
```suggestion
if [[ ("${target}" == x86_64-apple-darwin*) && ("${LLVM_MAJ_VER}" -ge "12") ]]; then
```
Sigh.